configs/mailer.ts

```diff
@@ -0,0 +1,19 @@
+import nodemailer from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+
+const host = process.env.EMAIL_SERVER_HOST;
+const email = process.env.EMAIL_SERVER_USER ?? '';
+const pass = process.env.EMAIL_SERVER_PASSWORD ?? '';
+const port = Number(process.env.EMAIL_SERVER_PORT) ?? 0;
+const transport: SMTPTransport.Options = {
+  host,
+  port,
+  secure: true,
+  // service: 'gmail',
```

Comment:
Never directly use gmail service, they will block our IP for any tiny mistake! 💀